src/SFA.DAS.Admin.Aan.Web/Controllers/ManageEvent/NetworkEventOrganiserNameController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using FluentValidation;
+using FluentValidation.AspNetCore;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Admin.Aan.Application.Services;
+using SFA.DAS.Admin.Aan.Web.Authentication;
+using SFA.DAS.Admin.Aan.Web.Infrastructure;
+using SFA.DAS.Admin.Aan.Web.Models.NetworkEvent;
+
+namespace SFA.DAS.Admin.Aan.Web.Controllers.ManageEvent;
+
+[Authorize(Roles = Roles.ManageEventsRole)]
+[Route("events/new/organiser", Name = RouteNames.ManageEvent.EventOrganiserName)]
+public class NetworkEventOrganiserNameController : Controller
+{
+
+    private readonly ISessionService _sessionService;
+    private readonly IValidator<EventOrganiserNameViewModel> _organiserNameValidator;
+
+    public NetworkEventOrganiserNameController(ISessionService sessionService, IValidator<EventOrganiserNameViewModel> organiserNameValidator)
+    {
+        _sessionService = sessionService;
+        _organiserNameValidator = organiserNameValidator;
+    }
+
+    public const string OrganiserNameViewPath = "~/Views/ManageEvent/EventOrganiserName.cshtml";
+
+
+    [HttpGet]
+    public IActionResult Get()
+    {
+        var sessionModel = _sessionService.Get<EventSessionModel>();
+        var augmentedModel = GetOrganiserNameViewModel(sessionModel);
+
+        return View(OrganiserNameViewPath, augmentedModel);
+    }
+
+    [HttpPost]
+    public IActionResult Post(EventOrganiserNameViewModel submitModel)
```

Comment:
FYI Since you have form tag around the full page and you are binding to full view model, the links and other values come prepopulated in the view model, so you don't have to hydrate the model again. However this is not the pattern we are following.